meson_options.txt

```diff
@@ -144,9 +144,9 @@ option('dbussessionservicedir', type : 'string',
 option('dbussystemservicedir', type : 'string',
        description : 'D-Bus system service directory')
 option('pkgconfigdatadir', type : 'string', value : '',
-       description : 'directory for ')
+       description : 'directory for arch-independent pkg-config files')
```

Comment:
`pkg-config files`?? Maybe, `arch-independent data files` is ok. WDYT?

Reply:
pkg-config manpage calls those files _"special metadata files"_, just `data files` is too generic for my taste, `library files` or `library metadata files` would imply it's for information related libraries only, which is not always the case and especially not here. So I just settled with `pkg-config files` for the lack of a good standardized term. `*.pc files` would be my second suggestion. However, I'm not going to bicker about something like this so make your final suggestion please and I'll force-push your re-wording.

Reply:
cc @keszybz.

Reply:
[Meson docs](https://mesonbuild.com/Pkg-config-files.html) refer to those files as "Pkg-config files", i.e. the same I did.

Reply:
Haha, I missed the option is about `pkgconfig`. Sorry, please forget my above comment.

LGTM.